include/manager.h

```diff
@@ -0,0 +1,69 @@
+#ifndef _MANAGER_H
+#define _MANAGER_H
+
+#include <assert.h>
+#include <string.h>
+#include <unistd.h>
+#include <dlfcn.h>
+#include <sys/auxv.h>
+
+// vDSO wrapper needed includes
+#include <sys/time.h>
+//#include <sys/vdso.h>
+
+/*******************************************************************************
+ * Compartment
+ ******************************************************************************/
+
+#define MAX_INTERCEPT_COUNT 4
+
+#include "compartment.h"
+
+#define ENV_FIELDS_CNT 1
```

Comment:
Not sure I know what `comp_env_fiesld` (or `ENV_FIELDS_CNT`) are.

Reply:
This is to do with stack setup on program launch, which I haven't completed, and that might need to be changed based on that recent Morello document [1]. Again, it's something I'd like to keep until a proper cleanup.

[1] https://git.morello-project.org/morello/kernel/linux/-/wikis/Morello-pure-capability-kernel-user-Linux-ABI-specification

Reply:
Would that be for this PR or a future PR?

Reply:
A future PR. Currently stuff is working in this regard, but it's not correct.